src/components/common/Modal/Modal.tsx

```diff
@@ -59,12 +57,16 @@ const Modal = ({
   const { t } = useTranslation();
   return (
     <Dialog open={isOpen} fullWidth={fullWidth} maxWidth={size}>
-      <ModalTitle>
-        {title}
-        <CloseButton onClick={() => handleClose()}>
-          <CloseIcon titleAccess={t('Close')} />
-        </CloseButton>
-      </ModalTitle>
+      <Grid container alignItems="center">
```

Comment:
Was having issues with the grids, so I went back to ```position: absolute```. But I figured out the issue. I was missing this ```alignItems="center"```